<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FVADictionaryEditorClass</class>
 <widget class="QDialog" name="FVADictionaryEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FVADictionaryEditor</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>441</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Добавить личность</string>
   </property>
   <widget class="QPushButton" name="btnAddPerson">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>22</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPerson">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Добавить место</string>
   </property>
   <widget class="QLineEdit" name="editPlace">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddPlace">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="fvadictionaryeditor.qrc"/>
 </resources>
 <connections/>
</ui>
